plugins/digitalocean/test-fixtures/config.yaml

```diff
@@ -0,0 +1,4 @@
+access-token: dop_v1_tr33mpd5m8q9t3ncisqbceydi8dd2n60pl1yiycg97z25fkqffp8j6ycjexample
+config: ${HOME}/Library/Application Support/doctl/config.yaml
+context: default
```

Comment:
Does the DigitalOcean config file support multiple contexts / profiles?

Reply:
The config file itself doesn't have a list of available contexts.
It can be found by using `doctl atuh list` command.

The `context` property in the config file represents the currently selected context.

Reply:
So should this importer then import those other `auth list` profiles too? (Can be in a different PR)

Reply:
Yeah, I think that is something we can achieve.
When we switch the context it populates the config file with the proper token value. So, we would need to read the content of the config for each candidate.

In general, the process would look like that.
- execute `doctl auth list` and retrieve all profiles.
- go through each of them:
           - execute `doctl auth swith --context <name>` to set current context.
           - read the config file and extract the token and profile name to create a candidate.

Reply:
But where does `doctl` get the token from when the user runs `doctl auth switch`?

Reply:
That's a good question. I suppose it calls some API under the hood to retrieve it.
At least `config.yaml` has a single `context` and `token` field that's replaced with different values when you do `switch`

Reply:
In that case it probably requires a user login to complete, which would be out of scope for (how we currently see) importers.